biscuit-auth/benches/token.rs

```diff
@@ -11,7 +11,7 @@ use rand::rngs::OsRng;
 
 fn create_block_1(b: &mut Bencher) {
     let mut rng = OsRng;
-    let root = KeyPair::new_with_rng(&mut rng);
+    let root = KeyPair::new_with_rng(Algorithm::Ed25519, &mut rng);
```

Comment:
it could be done in another PR, but adding benches for other signature algorithms would be nice (eg making the bench suite parametric on the algorithm, and running it for each supported algorithm)